src/consensus/params.h

```diff
@@ -48,6 +48,7 @@ struct Params {
     int nBudgetProposalEstablishingTime; // in seconds
     int nSuperblockStartBlock;
     int nSuperblockCycle; // in blocks
+    int nMinGovernanceQuorum; // Min absolute vote count to trigger an action 
```

Comment:
Could probably agree with min/minimum part (though for consistency with `nMasternodeMinimumConfirmations` I would prefer `Minimum`) but not sure we are on the same page with naming here (consensus params) in general. `Params` is a quite large struct already and it contains all kind of different info so imo it's better to keep its member's names structured like typeSomeClassSomeVariable.